Source/monstdat.h

```diff
@@ -89,44 +89,44 @@ enum class MonsterAvailability : uint8_t {
 };
 
 struct MonsterData {
-	const char *mName;
-	const char *GraphicType;
-	const char *sndfile;
-	const char *TransFile;
+	const char *name;
+	const char *graphicType;

```

Comment:
```suggestion
	const char *assetsSuffix;
```
That way it will have less conflicts with https://github.com/diasurgical/devilutionX/pull/5001 :)